src/main/java/mate/academy/model/impl/BookImpl.java

```diff
@@ -0,0 +1,49 @@
+package mate.academy.model.impl;
+
+import java.math.BigDecimal;
+import mate.academy.model.Book;
+
+public class BookImpl implements Book {
```

Comment:
Entity should not be an interface
```suggestion
public class Book {
```